server/chat-commands/core.ts

```diff
@@ -484,6 +484,13 @@ export const commands: ChatCommands = {
 		user.resetName();
 	},
 
+	nfb: 'nofeedback',
```

Comment:
```suggestion
```

I would not give an alias because this is not actually intended to be used by users.